client/i18n/locales/english/translations.json

```
@@ -569,7 +569,8 @@
     "intro": "Complete the following steps to claim and view your {{i18nCertText}}",
     "complete-project": "Complete {{i18nCertText}} Projects",
     "accept-honesty": "Accept our Academic Honesty Policy",
-    "set-name": "Set your name, and make it public",
+    "set-name-public": "Set your name settings to public",
```

```suggestion
    "set-name-public": "Set your name to be public",
```
tbh I'm not happy with either verbiage but can't think of something better at the moment